### PR TITLE
Add aarch64 wheel build support

### PR DESCRIPTION
**Pull request by odidev**
Add aarch64 wheel build support.
Related to https://github.com/pganssle/zoneinfo/issues/99  @pganssle Could you please review this PR?

### CODE REVIEW

**Comment by pganssle** [comment]
Oops, I will need to fix the tests here because of a change to some of the historical zones. Sorry about that. ☹

File: .github/workflows/build-publish.yml [hunk 1]
```diff
@@ -45,9 +45,13 @@ jobs:
         platform:
           - 'manylinux1_x86_64'
           - 'manylinux1_i686'
+          - 'manylinux2014_aarch64'
     name: Build a ${{ matrix.platform }} for ${{ matrix.python_tag }}
     steps:
       - uses: actions/checkout@v2
+      - uses: docker/setup-qemu-action@v1
```
Comment (pganssle) [inline .github/workflows/build-publish.yml:52]: I am assuming you know this better than me, but why is this necessary? Is there some documentation we can link in a comment here that explains why qemu is necessary to run the manylinux2014_aarch64 docker container?

Comment (odidev) [reply .github/workflows/build-publish.yml:52]: Github action provides x86_64 builders and not AArch64. Qemu is used to enable the execution of different multi-architecture containers. If we don’t use qemu, it will fail to run the AArch64 binaries on x86_64 machines.
If you have any further doubts please refer the [Link](https://github.com/multiarch/qemu-user-static)

**Comment by odidev** [comment]
@pganssle, could you please take a look into this PR? Thanks in advance.

**Review by pganssle — APPROVED** [review]

**Comment by kukat** [comment]
@pganssle could you please release the aarch64 package?

**Comment by karwal** [comment]
aarch64 still not available ?
